### DISCUSSION TITLE
Prysm on other Proof of Stake Chains

### DISCUSSION THREAD

**Discussion by Alphacolo** [OffchainLabs/prysm#10175] [General]
I'm using Prysm to validate for Ethereum Consensus Layer (ETH2).  Could Prysm work in a chain like Harmony One?

**Comment by rauljordan** [comment]
hi @Alphacolo unfortunately not. Prysm is an implementation of Ethereum proof-of-stake and only works for Ethereum. I'm pretty sure Harmony is an entirely different blockchain with a different model and specification
